### ISSUE TITLE
users = session.nearby_users() no longer returns a list

### ISSUE THREAD

**Issue by WhiskeyHammer** [charliewolf/pynder#143]
As per the README, session.nearby_users() should return a list.

_users = session.nearby_users() # returns a list of users nearby_

I started a new project using the latest version of Pynder (0.0.13) and now it returns <class 'generator'>.

_Try: print(type(session.nearby_users()))_

I tried the suggestion in https://github.com/charliewolf/pynder/issues/137 since it looked related, but I didn't see a change. 

How can I get session.nearby_users() to return a list again?

**Comment by mcfoton** [comment]
Same issue here, how to get around this?

**Comment by charliewolf** [comment]
The README should say `iterable`, not `list`. You should iterate through it as a generator. If you really want a list you can cast it, but that would be a waste of memory for most uses.

**Comment by WhiskeyHammer** [comment]
I don't think this is properly closed. The core issue is that the README demonstration, as written, does not work. You note it as an `iterable` then treat it as a list in the subsequent demo. The side issue is that code that's been written based on that demonstration has been invalidated by a change to `iterable`. 

I suggest:
1) Updating the README example to work with the new datatype 
2) Adding a method or an example that shows how to convert the iterable to a list so that legacy code will still work. 

I tried the below while researching this and neither worked for me. 

```
users = list(session.nearby_users()) 
#returns an empty list
```

```
users = session.nearby_users()
user = next(users) 
#returns StopIteration exception 
```

**Comment by WhiskeyHammer** [comment]
Aha! I have news! Both of the above options **do in fact work**. The issue was that the Tinder API failed to provide users for Pynder to work with. 

---**The Code Works** and using generators instead of lists was noticeably faster---
As part of my troubleshooting, I did a sanity check with the app on my phone, and it was stuck on 'Finding nearby users...' This can happen if you've run out of likes or if Tinder is having server or network problems. So the issue wasn't that the code wasn't working, it's that Tinder wasn't communicating back to the session on my phone or with the Pynder instance. I sent an email to support and was back up and running the next day. I was able to verify both of the above snippets, and I'll probably write an update for the README and throw a pull request sometime this weekend. Note that there was a noticeable speed difference when launching the program, between working with one generator at a time and casting them all to a list at once. 

---**Tinder Likely Suspends Accounts That Create Too Many Session**---
One more important piece of news for anyone who's following this. I'm a premium subscriber, so I shouldn't have run up against the like limit. I checked my session object in Pynder, and I had a very high number of remaining likes and I wasn't banned. What was unusual was that I had a huge negative value under `can_like_in`. This is the property that determines when you can send the next like. Running the numbers, the time ended up being something like a month+. I did some research on the negative number, and the general consensus in the Tinder API sub-community is that the negative number indicates that you've run out likes or rubbed up against some other Tinder limit (in either case Tinder won't bother sending you users to like). If anybody wants a source on that, let me know, and I'll dig it up from my history. Nobody else that I talked to seemed to be having issues communicating with the Tinder API, so I began to suspect that my account had been flagged as a bot... During testing I was rapidly creating, dropping, and recreating the sessions - this is obviously not the normal user pattern and, thinking as an anti-bot Tinder dev, it would be pretty trivial to automatically identify and suspend accounts that displayed this kind of session behavior. 

After I sent an email to support and my accounts started getting info back from Tinder, I began doing some testing. This _also_ involved a fair bit of session creating. Roughly one a minute or so for 10-15 minutes. Sure enough, Tinder eventually stopped returning requests, and my `can_like_in` property was negative and large. Admittedly, I didn't check the property while I was receiving responses buuuuut, the repeatable sequence of events, results, and obviously flaggable behavior are strong evidence to me that Tinder suspends accounts that create too many sessions too quickly. 

---**TL;DR**---
- The code snippets in the above comment work with the new version of Pynder 
- Tinder likely suspends accounts that create too many sessions too quickly

**Comment by VeNoMouS** [comment]
@WhiskeyHammer , did you notice that nearby_users() doesn't return any data if you hammer too much as well? do you know the time in which this comes back?
